dockerfiles/submitty/python/qiskit/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:22.04
+
+RUN apt-get update
+RUN apt-get install python3 pip -y
```

Comment:
We should pin the exact versions to ensure builds are as reproducible as possible

Reply:
I just specified the python and pip versions. Should I try to specify the qiskit package version as well?